houston/root_cause/root_cause.py

```diff
@@ -9,17 +10,21 @@
 from ..command import Command, Parameter
 from ..configuration import Configuration
 
-Domain = List[Tuple[int, Any, List[Parameter]]]
+Domain = List[Tuple[int, Type[Command], List[Parameter]]]
 
 
 class MissionDomain(object):
```

Comment:
Maybe use `attrs` to simplify this object?

Reply:
Yeah this class is awful and I'll probably just rewrite it. Later though.

Reply:
Sure.